mods/d2k/sequences/structures.yaml

```diff
@@ -1556,8 +1556,8 @@ starport.smuggler:
 		Frames: 643, 644, 645, 663, 664, 665
 		Length: 6
 		Offset: -16,-16
-	icon: DATA.R8 # TODO: blank
-		Start: 4281
+	icon: DATA.R8 # TODO This is same as Ordos one.
```

Comment:
If there is no reason to keep the `TODO`, you could remove it.

Reply:
We have a custom artwork for corrino palace, maybe someone may want to make one for Smuggler Starport and Mercenary HFac.